spec/railsapps/rails_41/config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += %w( email.css )
```

Comment:
Oops, that's right!
I missed it :<

Now I rebased without this change aa94e1ee7f0ee92b164f1e7137edbbeda1887c67